pkgs/development/compilers/flutter/flutter.nix

```
@@ -83,6 +83,11 @@ let
         cp -r . $out
         ln -sf ${dart} $out/bin/cache/dart-sdk
 
+        # Dart VSCode plugin specifically look for the following file in: 
```

```suggestion
        # Dart VS Code plugin specifically look for the following file in: 
```

https://code.visualstudio.com/brand#brand-name